consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/IbftBlockHeaderValidationRulesetFactoryTest.java

```diff
@@ -62,7 +62,7 @@ public void bftValidateHeaderPasses() {
         getPresetHeaderBuilder(2, proposerNodeKey, validators, parentHeader).buildHeader();
 
     final BlockHeaderValidator validator =
-        IbftBlockHeaderValidationRulesetFactory.blockHeaderValidator(5).build();
+        IbftBlockHeaderValidationRulesetFactory.blockHeaderValidator(5, Optional.empty()).build();
```

Comment:
nit: Can you add a method in the test to create the blockHeaderValidator, or maybe just a field variable